### ISSUE TITLE
[css-masking-1] `clip` applies-to section should mention <marker>, not <mask>

### ISSUE THREAD

**Issue by AmeliaBR** [w3c/fxtf-drafts#197]
The blue-box for the [`clip` property](https://drafts.fxtf.org/css-masking-1/#propdef-clip) currently says:

> Applies to:	Absolutely positioned elements. In SVG, it applies to elements which establish a new viewport, pattern elements and mask elements.

Pretty sure that `mask` should be `marker`, to match `overflow` and the [SVG 1.1 definition](https://www.w3.org/TR/SVG11/masking.html#OverflowAndClipProperties).  We've got `mask-clip` for clipping masks, after all.

**Comment by dirkschulze** [comment]
@AmeliaBR Agree. Did you check if implementations actually respect the `clip` property on those mentioned elements?

**Comment by AmeliaBR** [comment]
@dirkschulze Good question. No, I didn't check. Now I have. Results aren't pretty.

Test case: https://codepen.io/AmeliaBR/pen/5670978ec8f6904558d57aa6f66ca2de/

Tests run in EdgeHTML 16, Chrome 63, Firefox 58, and Safari 11.

Chrome, Safari, and Edge do not support `clip` on any elements with SVG layout. (They support it on absolutely positioned top-level `<svg>` elements, based on the CSS box layout requirements.)

Firefox supports `clip` for SVG, but inconsistently:

- a `clip` is applied on `<svg>`, `<image>`, and `<marker>`, but not on `<symbol>` or `<pattern>`

- I suspect that `<symbol>`  (which Firefox formerly implemented as a nested `<svg>` in the shadow DOM) would probably have been clipped prior to FF 56, but that was broken by the recent fixes to symbol style matching (it is now a separate element type, which must not have `clip` specifically turned on).

- To confuse things even more, a `clip` on a nested `<svg>` is applied in the parent coordinate system, but on a `<marker>` it gets applied in the marker's own `viewBox` coordinates.

______________________________________

At this point, I'd be tempted to just say that `clip` doesn't apply to elements that don't have CSS layout boxes.  After all, SVG has had `clip-path` for as long as it has had `clip`, so there isn't much legacy use case. But I don't know if there are any uses of it in non-browser SVG contexts. Maybe check with your own colleagues at Adobe and with @Tavmjong, for starters.

**Comment by dirkschulze** [comment]
@AmeliaBR I am sure that Adobe products behave inconsistent too. Thanks a lot for the test  cases! The results don’t really come to a surprise for me. I’ll bring it up with the WG but I currently follow your conclusion that we should consider to remove the SVG viewport creating elements. I’ll wait for Tav’s input and run the tests with Adobe Illustrator and Adobe Photoshop too.

**Comment by dirkschulze** [comment]
I'd propose to follow the suggestion from @AmeliaBR and just allow `clip` on elements with CSS layout boxes. It is deprecated anyway.
